examples/simple/etc/jupyter/jupyter_server_config.d/simple_ext.json

```diff
@@ -0,0 +1,8 @@
+{
+    "ServerApp": {
+        "jpserver_extensions": {
+            "simple_ext1": true,
+            "simple_ext2": true
```

Comment:
I would actually write two separate config files `simple_ext1.json` and `simple_ext2.json` here. 

This directory is typically used only by extensions to automatically enable/disable extensions—i.e. users shouldn't ever have to edit these files manually.

The most common pattern, then, is that every extension has its own JSON file. When the extension is enabled/disabled, the extension edits this JSON file. If two extensions are found in one file, you might get collision between extension enable/disable functions.